src/test/java/LoinPageTest.java

```diff
@@ -0,0 +1,81 @@
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.ForgotYourPasswordPage;
+import pages.LoginPage;
+
+import java.time.Duration;
+
+public class LoinPageTest {
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/drivers/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        driver.get("https://magento.softwaretestingboard.com/customer/account/login/");
+    }
+
+    @Test
+    public void loginPage_LeftPanel_HasTexDisplayedAndSignInButtonEnabled() {
```

Comment:
This test and test below should be merged into one. As they check the same element - login form

Reply:
It's not left panel, as it's not descriptive and code reader does not know what is a left panel. It should be LoginFormSection
loginPage_LoginFormSection_HasPageTitleIntroductionAndLoginFormCorrectlyDisplayed.

Reply:
Also as there should be only one test for checking on elements for Login form section I suggest to use soft assert.
It would look like that:
```SoftAssert softAssert = new SoftAssert();
        softAssert.assertTrue(loginPage.isTextPageTitleDisplayed());
        softAssert.assertTrue(loginPage.isTextRegisteredCustomerDisplayed());
        softAssert.assertTrue(loginPage.isTextSingInInstructionDisplayed());
        softAssert.assertTrue(loginPage.isTextEmailLabelDisplayed());
        softAssert.assertTrue(loginPage.isTextPasswordLabelDisplayed());
        softAssert.assertTrue(loginPage.isSignInButtonEnabled());
        softAssert.assertTrue(loginPage.isTextForgotYourPasswordDisplayed());
        softAssert.assertAll();```